app/src/main/java/org/mozilla/focus/browser/integration/BrowserMenuController.kt

```diff
@@ -21,7 +34,14 @@ class BrowserMenuController(
     private val openInCallback: () -> Unit,
     private val openInBrowser: () -> Unit
 ) {
+    @VisibleForTesting
+    private val currentTab: SessionState?
+        get() = store.state.findTabOrCustomTabOrSelectedTab(currentTabId)
+
+    @VisibleForTesting
+    internal var ioScope: CoroutineScope = CoroutineScope(Dispatchers.IO)
```

Comment:
It's a bit unfortunate that this scope never gets cancelled (so it's essentially a global scope?). But it looks like this controller has no idea what the lifetime of its menu is.